tools/jslint.js

```diff
@@ -13,9 +13,15 @@ const CLIEngine = require('./eslint').CLIEngine;
 const glob = require('./eslint/node_modules/glob');
 
 const cwd = process.cwd();
-const cli = new CLIEngine({
+const cliOptions = {
   rulePaths: rulesDirs
-});
+};
+
+// Check if we should fix errors that are fixable
+if (process.argv.indexOf('-F') !== -1)
```

Comment:
Or maybe put the result into above object, like `fix: process.argv.includes('-F')`, reducing the changes around here to a single line.